components/Route/RenderRoute.tsx

```diff
@@ -18,7 +18,7 @@ const RenderRoute = ({ route }: { route: Route }) => {
         belowLayerID="road-label"
         style={{
           lineColor: "#b8b8b8",
```

Comment:
```suggestion
          lineColor: foofTheme["color-basic-300"]
```
Ich mag die Linienfarbe, aber greif am besten direkt aufs Theme zu: